docs/quickstart.rst

```diff
@@ -135,6 +150,11 @@ A ``requirements.txt`` file is provided to install all required packages.
     # Install required packages
     pip install -r /path/to/sg-jira-bridge/requirements.txt
 
+.. note::
+    If you are upgrading from a previous version of the bridge, we recommend you re-install dependencies
+    as we've had to fork the ``jira`` Python module to add a missing feature. As of this writing, it
```

Comment:
I'm not keen on the "As of this writing". I'm not sure if that is correct grammar, but reading this, there is no way of knowing when this was written and therefore how likely this is up-to-date information. Is it worth stating the date?

Reply:
I've removed that comment. It doesn't bring any value to the user.